src/Utilities/functions.jl

```diff
@@ -1843,6 +1853,20 @@ function operate(
     return MOI.ScalarNonlinearFunction(Symbol(op), Any[f, g])
 end
 
+function operate(
+    ::typeof(-),
+    ::Type{T},
+    f::MOI.ScalarNonlinearFunction,
+) where {T}
+    if f.head == :- && length(f.args) == 1
+        # A simplification for -(-(f)) into f, but only if f is an SNF.
+        if f.args[1] isa MOI.ScalarNonlinearFunction
+            return f.args[1]
```

Comment:
This does not seem to be covered by the tests

Reply:
It is in the `GreaterToLess` bridge tests. Because it tries the `g = map_function(f)` and then `inverse_map_function(g)` to see if it's equal to `f`.

Reply:
All good then :)